src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -115,11 +116,17 @@ static void add_broker_config(
         }
     }
 
+    ss::sstring doc = include_documentation ? "doc string" : "";
```

Comment:
Use a fixed map with "key" -> "doc string" where doc string is a hardcoded value.

Reply:
>Use a fixed map with "key" -> "doc string" where doc string is a hardcoded value.

seems reasonable

Reply:
I'm using `base_property.desc()` instead.